### PR TITLE
Impl AsExpression and Queryable

### PR DESCRIPTION
**Pull request by quodlibetor**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -91,6 +134,24 @@ fn gen_from_sql(name: &syn::Ident, wrapped_ty: &syn::Ty) -> quote::Tokens {
 
 fn gen_from_sqlrow(name: &syn::Ident, wrapped_ty: &syn::Ty) -> quote::Tokens {
     quote! {
+        // #[cfg(not(feature = "unstable"))]
```
Comment (sgrif) [inline src/lib.rs]: I don't think you want to commit commented out code.

**Review by sgrif — APPROVED** [review]
I kinda glossed over the tests but this looks fine to me.

File: src/lib.rs (hunk 1 above)
Comment (quodlibetor) [reply src/lib.rs]: I do not.

**Comment by quodlibetor** [comment]
I've manually verified that this seems to work in Postgres, which is good enough for me for now.

**Pull request merged**